test/cookbooks/test/recipes/rubygems_url.rb

```diff
@@ -0,0 +1,7 @@
+raise 'This test is only for RHEL 6' unless platform_family?('rhel') && node['platform_version'].to_i == 6
```

Comment:
Any real reason to do this since we shouldn't be hitting this due to the kitchen config?

Reply:
No, cut and paste leftover. Removed.